backend/src/test/java/de/sambam/gamenighttracker/controller/GameControllerTest.java

```diff
@@ -53,6 +56,26 @@ public void addNewGameTest() {
         assertThat(postResponse.getStatusCode(), is(HttpStatus.OK));
         assertEquals(postResponse.getBody(), Game.builder().name("Scrabble").build());
         assertTrue(gameDb.getGameList().contains(Game.builder().name("Scrabble").build()));
+    }
+
+    @Test
+    @DisplayName("get request should list all games from gameDb")
+    public void listAllGamesFromDbTest() {
+        //GIVEN
+        this.gameDb.add(Game.builder().name("Cluedo").build());
+        this.gameDb.add(Game.builder().name("Mikado").build());
+
+        //WHEN
+        ResponseEntity<Game[]> getResponse = restTemplate.getForEntity(
+                "http://localhost:" + port + "/games", Game[].class);
+        Game[] allTheGamesinDb = getResponse.getBody();
+        Game expectedGame1 = Game.builder().name("Cluedo").build();
+        Game expectedGame2 = Game.builder().name("Mikado").build();
 
+        //THEN
+        assertThat(getResponse.getStatusCode(), is(HttpStatus.OK));
+        assertEquals(2, allTheGamesinDb.length);
+        assertEquals(expectedGame1, allTheGamesinDb[0]);
+        assertEquals(expectedGame2, allTheGamesinDb[1]);
```

Comment:
ok, but we can write this more elegantly using hamcrest matchers, maybe have a look at those.

Reply:
thanks. i have tried those but it did not work for me in this case... i will have another look after some more progress of the project.